v22.1/api-support-policy.md

```diff
@@ -0,0 +1,108 @@
+---
+title: API Support Policy
+summary: Learn about Cockroach Labs' policy for supporting CockroachDB APIs.
+toc: true
+docs_area: reference
+---
+
+Cockroach Labs exposes various application programming interfaces (APIs).
+
+The vast majority of changes to these interfaces are seamless additions of new functionality. However, some changes are backward-incompatible and may require you to adjust your integration. Changes to an API are introduced according to its *stability guarantees*, which depend on its *support policy*.
+
+This page includes the following information:
+
+- Our [support policies](#support-policies) for each type of API.
+- Our [stability guarantees](#stability-guarantees) for supported APIs.
+- A summary of [APIs](#apis) that CockroachDB makes available.
+
+## Support policies
+
+| Type             | Description                                                               | Stability guarantees |
+|------------------|---------------------------------------------------------------------------|----------------------|
+| Programmable     | Supported for interfacing with third-party automated tools.               | ✓                    |
+| Non-programmable | Supported for consumption by humans. Not supported for automation.        |                      |
+| Reserved         | Intended for use by CockroachDB developers. Not supported for public use. |                      |
+
+### Stability guarantees
```

Comment:
Do we want this nested under Support policies or at the same level? I'm unclear. If we bump it up to h2, then API changes and Versioning would get bumped up to h3s and show up in the toc.